.husky/pre-commit

```diff
@@ -1 +1 @@
-yarn lerna run --stream precommit && yarn generate-types && yarn lint-staged
+pnpm -r precommit && lint-staged
```

Comment:
The `precommit` script in `schema-to-ts/package.json` already executes `generate-types`. We don't need to run it twice, so I deleted `generate-types`.